### ISSUE TITLE
Conv3D - padding="same" does not work

### ISSUE THREAD

**Issue by SebastianPopescu** [keras-team/keras#7656]
Having an input mini batch of the form (None,30,35,30,1)

and applying the following layer:

Convolution3D(filters = 32, kernel_size = 5, strides = 5, padding="same",data_format='channels_last')(model_input)

I get a layer with shape of the form : (None,6,7,6,32)

As far as I understand by setting padding="same" the model should automatically zero-pad the input data to get a layer with a shape (None,30,35,30,32)

Am I doing something wrong? Or does Keras have an upper limit on the amount of zero-padding it can do? I assume the current issue is that that limit has been exceeded and it has resorted to the 'valid' type

**Comment by Drulludanni** [comment]
I am pretty new to Keras and convolutional neural networks, but I think the reason that padding="same"does not give you the same output shape is because strides > 1,  the padding only takes into account the kernel size.
